app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -118,7 +121,7 @@ class ImageRegionSelectionInteractionViewTest {
 
```

Comment:
You had to set the oppia test rule, else the RunOn annotation will not work.

```
  @get:Rule
  val oppiaTestRule = OppiaTestRule()
```

Reply:
`HelpFragmentTest` for reference

Reply:
I tested it without this Rule and I I wonder how it worked. Added it now it shouldn't break anywhere. Thanks for reference.